dm/config/subtask.go

```diff
@@ -348,7 +350,11 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 			return terror.ErrConfigLoaderDirInvalid.Delegate(err, c.LoaderConfig.Dir)
 		}
 		c.LoaderConfig.Dir = newDir
+	}
 
+	// adjust sorting dir
+	if c.Mode != ModeIncrement {
```

Comment:
also skip for ModeDump?

Reply:
addressed in https://github.com/pingcap/tiflow/pull/9228/commits/f18d7217d2a76a9ed00a6a8f3cfb4a9aedd97484